experiment/oss_fuzz_checkout.py

```diff
@@ -75,10 +75,14 @@ def _clone_oss_fuzz_repo():
     print(stderr)
 
 
-def clone_oss_fuzz(temp_repo: bool = True):
+def clone_oss_fuzz(oss_fuzz_dir: str = ''):
   """Clones the OSS-Fuzz repository."""
-  if temp_repo:
+  if not oss_fuzz_dir:
     _set_temp_oss_fuzz_repo()
+  else:
+    global OSS_FUZZ_DIR
+    OSS_FUZZ_DIR = oss_fuzz_dir
```

Comment:
Sorry I forgot to mention this nit: Swap the `if-else` block for better readability
```python
  if oss_fuzz_dir:
    global OSS_FUZZ_DIR
    OSS_FUZZ_DIR = oss_fuzz_dir
  else:
    _set_temp_oss_fuzz_repo()

```